CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2.10.0 / 2019-05-22
+
+* [FEATURE] Template expansion: Make external labels available as `$externalLabels` in alert and console template expansion. #5463
+* [FEATURE] Web UI: Sort alerts by group name. #5448
+* [FEATURE] TSDB: Add `prometheus_tsdb_wal_segment_current` metric for the WAL segment index that TSDB is currently writing to. tsdb#601
+* [FEATURE] Scrape: Add `scrape_series_added` per-scrape metric. #5546
+* [FEATURE] Log filesystem type on startup. #5558
+* [ENHANCEMENT] Discovery/kubernetes: Add labels `__meta_kubernetes_endpoint_node_name` and `__meta_kubernetes_endpoint_hostname`. #5571
+* [ENHANCEMENT] Discovery/azure: Add label `__meta_azure_machine_public_ip`. #5475
+* [ENHANCEMENT] TSDB: Simplify mergedPostings.Seek, resulting in better performance if there are many posting lists. tsdb#595
+* [ENHANCEMENT] Cmd/promtool: Use POST requests for Query and QueryRange. client_golang#557
+* [ENHANCEMENT] Web: Fixed Content-Type for js and css instead of using `/etc/mime.types`. #5551
+* [ENHANCEMENT] Web: Update to Bootstrap 4.3.1. #5506
```

Comment:
Not sure if we need to list this as there's no user-facing impact.

Reply:
I don't feel strongly. It looked to me that the UI was visually a bit different, and that might make people wondered what happened. But my impression might have been wrong anyway. Does anybody know if the Bootstrap update changed the visual appearance slightly?

Reply:
I didn't notice any visible change but I didn't look very carefully either.

Reply:
To get to the bottom of this, I did a side-by-side comparison. There is somehow a one-pixel move, but no real change in appearance. (That happened at an earlier version.)

I'll remove this line from the changelog.